comp/core/tagger/impl-remote/remote.go

```diff
@@ -186,8 +186,7 @@ func (t *remoteTagger) Start(ctx context.Context) error {
 
 	t.client = pb.NewAgentSecureClient(t.conn)
 
-	timeout := time.Duration(t.cfg.GetInt("remote_tagger_timeout_seconds")) * time.Second
-	err = t.startTaggerStream(timeout)
```

Comment:
I'm not sure this is going to work, this is just going to loop indefinitely while waiting for the core agent no ? It's not going to return early and let the rest continue